hbs.js

```diff
@@ -2,45 +2,66 @@ const hbs = require('handlebars')
 const path = require('path');
 
 const isArray = x => x.constructor === Array.prototype.constructor;
-const escape = x => x.replace(/\n/g, '<br />');
+const br = x => x.replace(/\n/g, '<br />');
+const escape = x => x;
```

Comment:
what's point of that?

Reply:
this is a placeholder for experiments now